nearcore/src/migrations.rs

```diff
@@ -68,6 +68,7 @@ fn apply_block_at_height(
         .apply_transactions(
             shard_id,
             &chunk_header.prev_state_root(),
+            None,
```

Comment:
This `None` is hard to read without referring to the function definition.

Don't know what's the best way to handle it in Rust. Leaving it as it is now is not a big deal.
In C++ I'd suggest writing `/*split_state_roots=*/None,`.

Reply:
Ignore this.
Clion already annotates function arguments at the callsite.